tests/test_store.py

```diff
@@ -506,6 +505,9 @@ def test_map_datetime_to_datetime(
     check_timestamp_lists(actual, expected)
 
 
+# TODO add test for index time
```

Comment:
Let me know if you want help finishing this or address mypy problems.

Reply:
Yes please!